core/evancore.vue.libs/src/components/table/table.vue

```diff
@@ -59,6 +76,7 @@ export default Component;
     &::-webkit-scrollbar-thumb {
       border-radius: 4px;
       background-color: rgba(0, 0, 0, 0.5);
+      box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
       -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
```

Comment:
not sure how it behaves within `::-webkit-` pseudo selectors, but browser prefixes should actually be added by sass, so `-webkit-box-shadow` may be removed ?